python/solid_dmft/dmft_cycle.py

```diff
@@ -781,7 +788,7 @@ def _dmft_step(sum_k, solvers, it, general_params,
                                         sum_k.spin_block_names[sum_k.SO])
     if general_params['calc_energies']:
         formatter.print_summary_energetics(observables)
-    if not general_params['csc'] and general_params['magnetic'] and sum_k.SO == 0:
```

Comment:
Why remove `sum_k.SO == 0` ? We need to ensure that we are performing a non SOC calculation. Otherwise the lines below will fail no?